src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+.carouselWrapper {
+  grid-column: 2;
+}
+
+@media (--tablet-and-down) {
+  .carouselWrapper {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-column: 1 / span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+}
+
+.slideQuote {
+  background: var(--color-gray-300);
```

Comment:
Could you add [`box-sizing: border-box;`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) to this style? Without it, the `height: 100%` ends up applying to just the content, but not the content + padding, which causes it to look like this:

<img width="1308" alt="Screen Shot 2020-11-09 at 10 49 14 PM" src="https://user-images.githubusercontent.com/1002748/98637822-d6b16980-22dd-11eb-9285-d9a326da1a81.png">


instead of:

<img width="1292" alt="Screen Shot 2020-11-09 at 10 49 22 PM" src="https://user-images.githubusercontent.com/1002748/98637830-da44f080-22dd-11eb-91fd-05a379b6620b.png">


```suggestion
  background: var(--color-gray-300);
  box-sizing: border-box;
```

Reply:
👍 Whoa! What is this magic? Thanks for introducing box-sizing property.

Reply:
Yeah, it's a bit magical. It's a property that changes the definition of "width" and "height". Some people turn it on for all elements, since they find it easier to style.